lib/aux_vis.cpp

```diff
@@ -320,7 +320,8 @@ void SetVisualizationScene(VisualizationScene * scene, int view,
 
    if (keys)
    {
-      SendKeySequence(keys);
+      // SendKeySequence(keys);
+      CallKeySequence(keys);
```

Comment:
It's hard to tell. It probably depends if you have "screenshot" keys in the sequence. The way the window updates are done now (just from the main loop, I think) this may not work even now.

Should be OK to just change it -- we'll fix any issues when we notice them.